modules/op-release-notes-1-1.adoc

```diff
@@ -0,0 +1,103 @@
+* The `tkn pipelinerun logs` command now displays logs for the final tasks listed in the `finally` section
+* Interactive mode support has now been provided to the `tkn task start` command and the `describe` subcommand for the following tkn resources:  `pipeline`, `pipelinerun`, `task`, `taskrun`, `clustertask`, and `pipelineresource`.
+* The `tkn version` command now displays the version of the triggers installed in the cluster.
+* The `tkn pipeline describe` command now displays parameter values and timeouts specified for Tasks used in the Pipeline.
+* Support added for the `--last` option for the `tkn pipelinerun describe` and the `tkn taskrun describe` commands to describe the most recent PipelineRun or TaskRun, respectively.
+* The `tkn pipeline describe` command now displays the conditions applicable to the Tasks in the Pipeline.
+* You can now use the `no-headers` and `all-namespaces` flags with the `tkn resource list` command.
+
+
+=== Triggers
+* The following Common Expression Language (CEL) functions are now added:
+** `parseURL`  to parse and extract portions of a URL
+** `parseJSON` to parse JSON value types embedded in a string in the `payload` field of the `deployment` webhook
+* A new interceptor for webhooks from Bitbucket has been added.
+* EventListeners now display the `Address URL` and the `Available status` as additional fields when listed with the `kubectl get` command.
+* TriggerTemplate params now use the `$(tt.params.<paramName>)` syntax instead of the `$(params.<paramName>)` to reduce the confusion between TriggerTemplate and ResourceTemplates params.
+* You can now add `tolerations` in the EventListener CRD to ensure that EventListeners will be deployed with the same configuration even if all nodes are tainted due to security or management issues.
+* You can now add a Readiness Probe for EventListener Deployment at `URL/live`.
+* Support for embedding `TriggerBinding` specifications in EventListener triggers.
+* Trigger resources are now annotated with the recommended `app.kubernetes.io` labels.
+
+== Deprecated features
+The following items are deprecated in this release:
+
+* The `--namespace` or `-n` flags for all cluster-wide commands, including the `clustertask` and `clustertriggerbinding` commands, are deprecated. It will be removed in the next release.
+* The `name` field in `triggers.bindings` within an EventListener has been deprecated in favor of the `ref` field and will be removed in a future release.
+* Variable interpolation in TriggerTemplates using ``$(params)` has been deprecated in favor of using `$(tt.params)` to reduce confusion with the Pipeline variable interpolation syntax. The `$(params.<paramName>)` syntax will be removed in the next release.
+* The `tekton.dev/task` label is deprecated on `ClusterTasks`.
+* The `TaskRun.Status.ResourceResults.ResourceRef` field is deprecated and will be removed.
+* The `tkn create -f` command has been deprecated.
```

Comment:
@khrm did the create command exist? Wasn't it just a subcommand?

Reply:
I don't remember but yes.

Reply:
@pradeepitm12 Can you clarify here?

Reply:
It was there for a short period of time.